releasenotes/notes/0.9/improvements-in-DAG-drawer-73283f18ba5ba937.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The DAG drawer was enhance in the context of Jupyter notebooks. The parameter filename is now
```

Comment:
```suggestion
    The DAG drawer was enhanced in the context of Jupyter notebooks. The parameter filename is now
```

Reply:
Where was this introduced? I don't see any changes to the `qiskit.visualization.dag_drawer()` since 0.8. The only commit in the log that touches `qiskit.visualization.dag_visualization` was #2328 which just changed the error message if the dependencies weren't installed.